nyc_taxi/nyc_taxi.ipynb

```diff
@@ -428,9 +446,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "colors = [\"#FF0000\",\"#FF3F00\",\"#FF7F00\",\"#FFBF00\",\"#FFFF00\",\"#BFFF00\",\"#7FFF00\",\"#3FFF00\",\n",
-    "          \"#00FF00\",\"#00FF3F\",\"#00FF7F\",\"#00FFBF\",\"#00FFFF\",\"#00BFFF\",\"#007FFF\",\"#003FFF\",\n",
-    "          \"#0000FF\",\"#3F00FF\",\"#7F00FF\",\"#BF00FF\",\"#FF00FF\",\"#FF00BF\",\"#FF007F\",\"#FF003F\"]"
+    "colormap = cc.CET_C1s\n",
+    "num_hours = 24\n",
+    "indices = np.linspace(0, len(colormap) - 1, num_hours, dtype=int)\n",
+    "colors = [colormap[i] for i in indices]"
```

Comment:
<img width="1040" alt="image" src="https://github.com/holoviz-topics/examples/assets/35924738/e588d473-684e-404e-a479-3b9fbbae0abf">

Reply:
Maybe with `cc.CET_C7s`?

<img width="914" alt="image" src="https://github.com/holoviz-topics/examples/assets/35924738/8eb6698b-549e-421d-a7ac-204da0e9df9e">

You can easily display the cyclic color maps with:

```python
from colorcet.plotting import swatches
import holoviews as hv
hv.extension('bokeh')

swatches(group=['cyclic'])
```

Reply:
Thanks. I'll admit here that I'm not so good with color selection hence why I asked in the discord channel previously if the colors I selected were OK. I was more interested in using the colormaps available via `colorcet` rather than the old method of just listing the colors manually.

Reply:
I've used `CET_C7s` now